### DISCUSSION TITLE
No data available for Single Select [May 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2184] [Discord Threads]
> 🙋‍♂️ **alexjj**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### see https://github.com/nocodb/nocodb/issues/2059  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/976596318514470952)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
